src/user/USER_TIME_PLAYED_URL.ts

```diff
@@ -0,0 +1,2 @@
+export const USER_TIME_PLAYED_URL =
```

Comment:
It seems to me this isn't a users-related URL. It's a gamelist-related URL. I think all of this new code should live under a new _src/gamelist_ directory. Then this variable can be renamed to `GAMELIST_BASE_URL`:

```ts
export const GAMELIST_BASE_URL = "https://m.np.playstation.com/api/gamelist";
```

The function importing this can tack on the other stuff, like:

```ts
const url = buildRequestUrl(GAMELIST_BASE_URL, "/v2/users", ...);
```

Reply:
Yeah, that can be done on my next pass.